src/components/BrowserApp.tsx

```
@@ -0,0 +1,343 @@
+      {/* Viewport — captures all input */}
+      <div
+        ref={viewportRef}
+        className="flex-1 relative overflow-hidden bg-black outline-none"
+        tabIndex={0}
+        onClick={handleClick}
+        onDoubleClick={handleDoubleClick}
+        onWheel={handleWheel}
+        onKeyDown={handleKeyDown}
+        onMouseMove={handleMouseMove}
+        onContextMenu={(e) => e.preventDefault()}
+      >
```

Biome (2.4.6)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)